test/suites/pki.sh

```diff
@@ -189,9 +189,8 @@ test_pki() {
     # Remove remote
     lxc_remote remote remove pki-lxd
 
-    # Add the remote again using an incorrect token.
-    # This should succeed as is the same as the test above but with an incorrect token rather than no token.
-    lxc_remote remote add pki-lxd "${LXD5_ADDR}" --accept-certificate --token=bar
+    # Ensure remote can be added without the token, as we are already trusted by the remote.
+    echo "y" | lxc_remote remote add pki-lxd "${LXD5_ADDR}"
```

Comment:
The remote still trusts the client but the client deleted the `remote.crt` so it has to prompt for `y/fingerprint` before proceeding.

Manually `echo`'ing `y` feel like emulating `--accept-certificate`.

Reply:
Correct, should use `--accept-certificate` instead.